### ISSUE TITLE
Add option to generate app without SSR ~3

### ISSUE THREAD

**Issue by daffl** [donejs/donejs#549]
This came up many times before (latest https://github.com/donejs/donejs/issues/519) and although you can just open `development.html` it is fairly easy to add to the generator. It should ask if you would like server side rendering and if not, disable pushstate routing, start development mode with the new `--static` flag and add an `index.html` file for production.

**Comment by phillipskevin** [comment]
Will there then be a `donejs add ssr` for people who change their mind?

**Comment by daffl** [comment]
I've been thinking more about the `donejs add` functionality and it should be possible. I'd like to have things like `donejs add mocha` as well but that definitely will require some changes.

**Comment by justinbmeyer** [comment]
I don't think we should ask people this.  I think we ask them too many package.json questions already.

**Comment by daffl** [comment]
If we'd make a DoneJS FAQ this is definitely the most commonly asked question. I'd rather throw out some of the `package.json` questions.

**Comment by justinbmeyer** [comment]
It still shouldn't be a config. We already produce the HTML files.

**Comment by pYr0x** [comment]
i am with david.
we should cover that user request!
an options makes sense for me.

the world is not only SSR.

**Comment by justinbmeyer** [comment]
@pYr0x 

> the world is not only SSR.

It should be :-).  It's a big part of the DoneJS sales pitch.  

@daffl it's possible someone would still want pushstate routing.  Could we setup a server to serve up the right `index.html` so it points to steal correctly?  My guess is that people would replace this eventually, but it might be a useful example of how to make pushstate work.
